compiler/rustc_span/src/symbol.rs

```diff
@@ -681,6 +681,7 @@ symbols! {
         lateout,
         lazy_normalization_consts,
         le,
+        len,
```

Comment:
Since the addition of the `slice_len_fn` lang item, this is now unused, right ?

```suggestion
```

Reply:
I believe it was a source of a performance gain in the first performance CI test: I was using a combination of `slice_impl` + `len` to find out a function, that is not optimal, but in some tests performance gains were substantial and I believe it may have helped. In any case, it's quite common expression and it may be good to intern in any case